Source/Scene/QuadtreePrimitive.js

```diff
@@ -754,17 +1009,28 @@ define([
         var endTime = getTimestamp() + primitive._loadQueueTimeSlice;
         var tileProvider = primitive._tileProvider;
 
-        processSinglePriorityLoadQueue(primitive, frameState, tileProvider, endTime, tileLoadQueueHigh);
-        processSinglePriorityLoadQueue(primitive, frameState, tileProvider, endTime, tileLoadQueueMedium);
-        processSinglePriorityLoadQueue(primitive, frameState, tileProvider, endTime, tileLoadQueueLow);
+        var didSomething = processSinglePriorityLoadQueue(primitive, frameState, tileProvider, endTime, tileLoadQueueHigh, false);
+        didSomething = processSinglePriorityLoadQueue(primitive, frameState, tileProvider, endTime, tileLoadQueueMedium, didSomething);
```

Comment:
`didSomething` is a bit vague. Maybe it should be something like `loadedTile` ?

Reply:
Changed it to `didSomeLoading`. The idea is to make sure that at least one tile gets a chance to make some load progress. Previously one tile per priority group was given a chance to load, which was less than ideal.